### PR TITLE
Migrate to elements in repo

### LINKED ISSUE #324: Move custom elements to their own repository.
This enables us to reuse them for our lancie-admin.
Requires #271

### PR DESCRIPTION
**Pull request by martijnjanssen**
Fixes #324

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -12,7 +12,9 @@
     "app-layout": "PolymerElements/app-layout#^0.10.4",
     "app-route": "polymerelements/app-route#^0.9.2",
     "app-storage": "PolymerElements/app-storage#^0.9.8",
-    "iron-lazy-pages": "TimvdLippe/iron-lazy-pages#2.0-preview"
+    "iron-lazy-pages": "TimvdLippe/iron-lazy-pages#2.0-preview",
+    "lancie-dialog":"AreaFiftyLAN/lancie-dialog",
+    "lancie-ajax":"AreaFiftyLAN/lancie-ajax"
```
Comment (TimvdLippe) [inline bower.json]: Specify the versions

**Comment by MatthijsKok** [comment]
Fixes #444

**Comment by FrancisBehnen** [comment]
Pitty reviews aren't posted on slack

File: src/lancie-my-area/my-area-profile.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
 <link rel="import" href="../../bower_components/paper-tooltip/paper-tooltip.html">
 <link rel="import" href="../../bower_components/paper-toast/paper-toast.html">
 
-<link rel="import" href="../lancie-login/lancie-ajax.html">
+<link rel="import" href="../../bower_components/lancie-ajax/lancie-ajax.html">
 <link rel="import" href="../lancie-login/lancie-passwords.html">
```
Comment (martijnjanssen) [inline src/lancie-my-area/my-area-profile.html]: This is the problem with the double register, the `lancie-passwords` element has to be removed from this repo. Import the `lancie-passwords` element from the lancie-login-form in the bower_components.

**Comment by martijnjanssen** [comment]
Looks good! Waiting on the build before merging.

**Pull request merged**